src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/ContactSaveRequest.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Requests;
+
+public class ContactSaveRequest
+{
+    [Required] public string TeamName { get; set; } = string.Empty;
+
+    [Required, EmailAddress] public string TeamEmail { get; set; } = string.Empty;
+
+    [Required] public string ContactName { get; set; } = string.Empty;
+
+    public string? ContactTelNo { get; set; }
```

Comment:
You mean frontend-only validation right? 

Whilst it's not super likely to see anyone trying to circumvent this, it's not really good practice to skip server-side valiation. 

It's fairly straightforward to create custom validations so I'd suggest we don't skip our belts and braces. 

We should at least create a new `PhoneNumber` validation attribute, and add the built-in `MinLength` validation. I'd probably say we don't desperately need the validation for the DfE phone number, but up to you.

Reply:
> You mean frontend-only validation right?

Sorry yeah frontend-only validation.

I'm not keen on implementing this just because I don't feel like it's worth the time adding. If an analyst were to get around the validation, it would be simple to change after the fact. So even if an analyst got around the validation, I don't think it's a big deal.

If you're keen on it, maybe take it up with Cam?

Reply:
Added validation after our discussion with Cam.

I've not used a new attribute, but `ContactSaveRequest` now inherits `IValidatableObject`. I didn't use `MinLength` as `ContactTelNo` can be null - I just put all the validation into the `Validate` method. Also added unit tests in `ContactSaveRequestTests.cs`.